sources/models/defaults.toml

```diff
@@ -92,3 +92,8 @@ template-path = "/usr/share/templates/chrony-conf"
 
 [metadata.settings.ntp]
 affected-services = ["chronyd"]
+
+# Kubernetes
+
+[settings.kubernetes]
+cluster-domain = "cluster.local"
```

Comment:
Kubernetes settings need to go into the Kubernetes override files, rather than this file for all variants, because they don't apply to all variants.  (If this is here and you try a non-Kubernetes variant (currently just `aws-dev`) you'll see the API fail to deserialize the data store.)

Currently that means `src/aws-k8s-1.15/override-defaults.toml`, since 1.16 and 1.17 models are symlinked.  Would you mind moving this there?

Reply:
Ah. I was only thinking about the kubernetes case. I'll move the default setting.